go/vt/schemadiff/errors.go

```diff
@@ -294,8 +294,23 @@ type ForeignKeyLoopError struct {
 func (e *ForeignKeyLoopError) Error() string {
```

Comment:
@shlomi-noach Should we change `ForeignKeyLoopError` to have a different `Loop` type? So something like this maybe:

```go
type ForeignKeyColumn struct {
	Table  string
	Column string
}

type ForeignKeyLoopError struct {
	Table string
	Loop  []ForeignKeyColumn
}
```

This because we now better track this at a column level, so this way we have the details of what the loop looks like?

Reply:
Good point. I'll do so.

Reply:
Done in https://github.com/vitessio/vitess/pull/15431/commits/d3b8c8e2b1777877bb1e2297d39b2f8dd9773331